### PR TITLE
[macAging]:asic mac aging should trigger asicDB mac remove

### PR DESCRIPTION
**Pull request by gongjianLhr**
Signed-off-by: Haoran Liu gongjian.lhr@alibaba-inc.com

### CODE REVIEW

File: syncd/syncd_notifications.cpp [hunk 1]
```diff
@@ -78,7 +78,15 @@ void redisPutFdbEntryToAsicView(
     std::string strFdbEntry = sai_serialize_fdb_entry(fdb->fdb_entry);
 
     std::string key = ASIC_STATE_TABLE + (":" + strObjectType + ":" + strFdbEntry);
-
+	if((fdb->event_type == SAI_FDB_EVENT_AGED)||(fdb->event_type == SAI_FDB_EVENT_FLUSHED))
```
Comment (kcudnik) [inline syncd/syncd_notifications.cpp]: please fix ident and add { } to each IF

Comment (gongjianLhr) [reply syncd/syncd_notifications.cpp]: Hi Kcudnik, Thanks for review, have fixed the problems you mentioned.

**Comment by stcheng** [comment]
@kcudnik take a look?

**Comment by stcheng** [comment]
hello @gongjianLhr could you please resolve the conflicts? we will evaluate this pull request later.

**Comment by gongjianLhr** [comment]
Hi @stcheng ,conflicts resolved

File: syncd/syncd_notifications.cpp [hunk 2]
```diff
@@ -83,7 +83,7 @@ void redisPutFdbEntryToAsicView(
     std::string strFdbEntry = sai_serialize_fdb_entry(fdb->fdb_entry);
 
     std::string key = ASIC_STATE_TABLE + (":" + strObjectType + ":" + strFdbEntry);
-
+
```
Comment (stcheng) [inline syncd/syncd_notifications.cpp]: extra spaces?

File: syncd/syncd_notifications.cpp [hunk 3]
```diff
@@ -92,6 +92,19 @@ void redisPutFdbEntryToAsicView(
         return;
     }
 
+    if((fdb->event_type == SAI_FDB_EVENT_AGED)||(fdb->event_type == SAI_FDB_EVENT_FLUSHED))
```
Comment (stcheng) [inline syncd/syncd_notifications.cpp]: could you add one space between `if` and `(`?

Comment (stcheng) [reply syncd/syncd_notifications.cpp]: same as below

**Comment by gongjianLhr** [comment]
Hi @stcheng misc issues resolved

**Pull request merged**